Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelSelectTypePage.xaml.cs

```diff
@@ -302,7 +302,7 @@ private void BrowseButtonClicked(object sender, RoutedEventArgs e)
                     {
                         System.Windows.Forms.OpenFileDialog dlg = new System.Windows.Forms.OpenFileDialog();
 
-                        dlg.Filter = "XML Files (*.xml)|*.xml" + "|WSDL Files (*.wsdl)|*.wsdl" + "|All Files (*.*)|*.*";
+                        dlg.Filter = "WSDL Files (*.wsdl)|*.wsdl" + "|XML Files (*.xml)|*.xml" + "|All Files (*.*)|*.*";
 
```

Comment:
what about Swagger files JSON/YAML, and only JSON- don't they also need filter type?